frontend/packages/console-shared/src/components/popover-body/popoverBody.scss

```diff
@@ -0,0 +1,55 @@
+.ocs-popover{
+    min-width:250px;
+    max-width:250px;
+}
+
+.ocs-popover__header{
+    font-weight: 800;
+    font-style: bold;
+}
+
+.ocs-popover__dropdown{
+    min-width:250px;
+    max-width: 250px;
+    line-height:25px;
+    margin-top:2px;
+    margin-bottom: 2px;
+    padding:2px;
+    font-size:1.01em !important;
+}
+
+.ocs-popover-items{
+    margin-top:5px;
+    min-width: inherit;
+    max-width: 250px;
+    display: grid;
+    grid-template-columns: [object]70% [value]30%;
+    grid-column-gap: 5px;
+    grid-row-gap:3px;
+    grid-template-rows:20% 20% 20% 20% 20%;
```

Comment:
Nit: it should be possible to avoid manual repetition via `repeat()` expression [1].

[1] https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-rows